backend/tracim_backend/lib/utils/authentication.py

```diff
@@ -56,23 +58,38 @@ def __init__(self, app_config: CFG, configurator: Configurator) -> None:
             config_data = json.load(config_file)
             self.saml_config = Saml2Config()
             self.saml_config.load(config_data)
-            attribute_map = {
-                "EmailAddress": "user_email",
-                "UserID": "user_id",
-                "FirstName": "first_name",
-                "LastName": "last_name",
-            }
-            self.saml_config.allow_unknown_attributes = True
-            self.saml_config.attribute_map = attribute_map
         self._metadata_response = Response(content_type="application/xml")
         self._metadata_response.text = create_metadata_string(
             config=self.saml_config, configfile=None
         ).decode()
         self.saml_client = Saml2Client(config=self.saml_config)
 
-        for _, data in self.saml_client.config.vorg.items():
+        self.saml_idp_config = {
+            "default": {
```

Comment:
move this dict to a global static var on top of authentication.py file: `SAML_IDP_DEFAULT_CONFIG = { ... }`

Reply:
done